### PR TITLE
add info in README to help building

### PR DESCRIPTION
**Pull request by Issif**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+```
+
+#### Test in local
+
+```bash
+make server
+```
+You can access your local website on http://localhost:1313
+
+#### Build
```
Comment (lucperkins) [inline README.md]: This command is only for publishing the site. It doesn't need to be included here.

Comment (Issif) [reply README.md]: As it was already included in Makefile, I added it. Will remove it.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+#### Run hugo server
+
+```bash
+make server
+```
+You can access your local website on http://localhost:1313
+
+## Translations
+
+### Chineses
```
Comment (lucperkins) [inline README.md]: Chinese

File: README.md [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+
+```bash
+cd ./falco-website/themes/falco-fresh
+yarn install
+```
+
+#### Run hugo server
+
+```bash
+make server
```
Comment (lucperkins) [inline README.md]: make serve

File: README.md [hunk 4]
```diff
@@ -1,12 +1,44 @@
...
+
+### Test locally
+
+#### Clone repository
+
+```bash
+git clone git@github.com:falcosecurity/falco-website.git
+```
+
+#### Install theme
```
Comment (lucperkins) [inline README.md]: This step is unnecessary, as it's part of `make serve`

File: README.md [hunk 5]
```diff
@@ -1,12 +1,44 @@
 # falco-website
 
-## Translation
+## Prerequisites
 
-There is currently scaffolding in place for a Chinese translation of the site. To develop the translation, run `make serve`. This will start the Hugo server running locally. Navigate to `http://localhost:1313/zh` to see the current Chinese translation.
+To build this website you need :
+
+* [hugo](https://gohugo.io/getting-started/installing/)
+* [nodejs](https://nodejs.org/en/download/)
```
Comment (lucperkins) [inline README.md]: Node.js

File: README.md [hunk 6]
```diff
@@ -1,12 +1,44 @@
 # falco-website
 
-## Translation
+## Prerequisites
 
-There is currently scaffolding in place for a Chinese translation of the site. To develop the translation, run `make serve`. This will start the Hugo server running locally. Navigate to `http://localhost:1313/zh` to see the current Chinese translation.
+To build this website you need :
+
+* [hugo](https://gohugo.io/getting-started/installing/)
```
Comment (lucperkins) [inline README.md]: Capitalize Hugo and Yarn

File: README.md [hunk 7]
```diff
@@ -1,12 +1,44 @@
 # falco-website
 
-## Translation
+## Prerequisites
 
-There is currently scaffolding in place for a Chinese translation of the site. To develop the translation, run `make serve`. This will start the Hugo server running locally. Navigate to `http://localhost:1313/zh` to see the current Chinese translation.
+To build this website you need :
```
Comment (leodido) [inline README.md]:
```suggestion
To build this website you need:
```

File: README.md [hunk 8]
```diff
@@ -1,12 +1,44 @@
...
+cd ./falco-website/themes/falco-fresh
+yarn install
+```
+
+#### Run hugo server
+
+```bash
+make server
+```
+You can access your local website on http://localhost:1313
```
Comment (leodido) [inline README.md]:
```suggestion

You can access your local website on http://localhost:1313
```

**Pull request merged**